supertokens_python/recipe/thirdpartypasswordless/api/implementation.py

```diff
@@ -101,36 +104,17 @@ async def consume_code_post(self,
                                 device_id: Union[str, None],
                                 link_code: Union[str, None],
                                 api_options: PasswordlessAPIOptions,
-                                user_context: Dict[str, Any]) -> ConsumeCodePostResponse:
-        otherType = await self.pless_consume_code_post(pre_auth_session_id, user_input_code, device_id, link_code, api_options, user_context)
-        if otherType.is_ok:
-            if otherType.created_new_user is None or otherType.user is None or otherType.session is None:
-                raise Exception("Should never come here")
-            return ConsumeCodePostOkResponse(otherType.created_new_user, User(otherType.user.user_id, otherType.user.email, otherType.user.phone_number, None, otherType.user.time_joined), otherType.session)
-        if otherType.is_expired_user_input_code_error:
-            if otherType.failed_code_input_attempt_count is None or otherType.maximum_code_input_attempts is None:
-                raise Exception("Should never come here")
-            return ConsumeCodePostExpiredUserInputCodeErrorResponse(otherType.failed_code_input_attempt_count, otherType.maximum_code_input_attempts)
-        if otherType.is_general_error:
-            if otherType.message is None:
-                raise Exception("Should never come here")
-            return ConsumeCodePostGeneralErrorResponse(otherType.message)
-        if otherType.is_incorrect_user_input_code_error:
-            if otherType.failed_code_input_attempt_count is None or otherType.maximum_code_input_attempts is None:
-                raise Exception("Should never come here")
-            return ConsumeCodePostIncorrectUserInputCodeErrorResponse(otherType.failed_code_input_attempt_count, otherType.maximum_code_input_attempts)
-
-        # restart flow error
-        return ConsumeCodePostRestartFlowErrorResponse()
+                                user_context: Dict[str, Any]) -> Union[ConsumeCodePostOkResponse, ConsumeCodePostRestartFlowErrorResponse, ConsumeCodePostGeneralErrorResponse, ConsumeCodePostIncorrectUserInputCodeErrorResponse, ConsumeCodePostExpiredUserInputCodeErrorResponse]:
+        return await self.pless_consume_code_post(pre_auth_session_id, user_input_code, device_id, link_code, api_options, user_context)
```

Comment:
By doing this, you are changing the return type to be from another recipe. Don't do that. Undo this change (in this PR and in the base branch PR - thirdpartyemailpassword recipe)